### ISSUE TITLE
[Bug Report][3.6.13] v-file-input rules uses Array value even when `multiple` prop is false or not specified

### ISSUE THREAD

**Issue by Paul-DS** [vuetifyjs/vuetify#20247]
### Environment
**Vuetify Version:** 3.6.13
**Vue Version:** 3.4.27
**Browsers:** Chrome 127.0.0.0
**OS:** Linux x86_64

### Steps to reproduce
Create a v-file-input without `multiple` property. Add a validation rule.

### Expected Behavior
The value passed to validation rules should be a `File` or `null` (to be consistent with the model value)

### Actual Behavior
The value passed to validation rules is an array

### Reproduction Link
[https://play.vuetifyjs.com/#...](https://play.vuetifyjs.com/#eNpVUMtuwjAQ/JWVVYkiJU6rHiqFgFrxByC4AAcTFohYHMuPXBD/zjoQCCd7Z2ZnNbO6CGfL7N8Y2QQUuSg8ng0pj5O1BiiaVBnTftuhrLVXlUb7gACmRyxP4I8IzLmaEKg+uI7llX1FmFbaBN+BAKS2SOO1mFf6wBtRAneJeIlyGwgdq1aNIhhP4NKdkHziczCLNDAXcJDEdzgCiz5YDXtFDkdw3fT9/oLZca78XO+QlnGNvd8cF63gafmBDWo/fHk8QxdZP1dXD4Pv/UTkXl+R9WoV10QHokT8yF/5lW7RK/ktNjdFGnzo)

### Other comments
I suppose #17687 fixed the same behavior on the model, but not on the rules

**Comment by wyhsunflower** [comment]
Hi, I already work on this issue and will raise PR later.

**Comment by RoryPTB** [comment]
I can also reproduce this issue, but I've noticed that this **only occurs in development mode** (`npm run dev`). It behaves correctly when the application is built and previewed. This awkwardly forces you to write two conditions depending on the mode the application is run in.

**Comment by eeriksen** [comment]
The model is still returning an empty array even though no `multiple` is specified (3.7.2).

**Comment by its-bru** [comment]
Perhaps a question for @johnleider. While on the surface this appears like a 'bug' its more like a 'change' in requirements. For those of us that have been using this component for months (July of 24 for me) this 'fix' became a 'breaking' change.

Moving forward I guess, what can we do differently, can this type of change be disseminated in some fashion?

**Comment by johnleider** [comment]
> Perhaps a question for [@johnleider](https://github.com/johnleider?rgh-link-date=2025-03-19T12%3A21%3A03.000Z). While on the surface this appears like a 'bug' its more like a 'change' in requirements. For those of us that have been using this component for months (July of 24 for me) this 'fix' became a 'breaking' change.
> 
> Moving forward I guess, what can we do differently, can this type of change be disseminated in some fashion?

I'm sorry to hear that. Definitely not intended. When something similar has happened in the past and we received a lot of reports, we would update the release notes with some information regarding the subject.

**Comment by Paul-DS** [comment]
> Perhaps a question for [@johnleider](https://github.com/johnleider). While on the surface this appears like a 'bug' its more like a 'change' in requirements. For those of us that have been using this component for months (July of 24 for me) this 'fix' became a 'breaking' change.
> 
> Moving forward I guess, what can we do differently, can this type of change be disseminated in some fashion?

Yes I agree, I suppose this should have been mentionned as "breaking change" in the release notes for [v3.7.13](https://github.com/vuetifyjs/vuetify/releases/tag/v3.7.13)
